packages/design-system-docs/src/pages/components/UsaBanner/UsaBanner.example.jsx

```diff
@@ -2,4 +2,12 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { UsaBanner } from '@cmsgov/design-system';
 
-ReactDOM.render(<UsaBanner />, document.getElementById('js-example'));
+ReactDOM.render(
+  <div>
+    <h6 className="preview__label">.gov Domains</h6>
```

Comment:
Maybe these labels can just be generalized to be 
`English banner` and `Spanish banner`